api/src/main/java/org/openmrs/module/serialization/xstream/converter/CustomCGLIBEnhancedConverter.java

```diff
@@ -27,6 +26,7 @@
 import com.thoughtworks.xstream.io.HierarchicalStreamWriter;
 import com.thoughtworks.xstream.mapper.CGLIBMapper;
 import com.thoughtworks.xstream.mapper.Mapper;
+import org.springframework.cglib.proxy.Enhancer;

```

Comment:
Shouldn't this import be ordered with the other `org` imports?

Reply:
@ibacher, I updated the PR according to this. However, there are few places in our code base that do not follow this:

https://github.com/openmrs/openmrs-module-serialization.xstream/blob/87f07a9e0e1550b6439826e12079ee9a9a7a58a2/api/src/main/java/org/openmrs/module/serialization/xstream/XStreamSerializer.java#L63-L64

https://github.com/openmrs/openmrs-module-serialization.xstream/blob/87f07a9e0e1550b6439826e12079ee9a9a7a58a2/api/src/main/java/org/openmrs/module/serialization/xstream/converter/HibernateCollectionConverter.java#L3-L13

Reply:
We should eventually fix those. Technically the import order is part of our coding standards, but it's not well-enforced across projects.